overlay/ports/fdk-aac/portfile.cmake

```diff
@@ -1,3 +1,7 @@
+# build the dynamic only so the user can use either the stripped or unstripped 
+# version according to his patent or license rights
+vcpkg_check_linkage(ONLY_DYNAMIC_LIBRARY ONLY_DYNAMIC_CRT)
```

Comment:
I don't think this is the right way to handle this and I doubt it will work either. I think we need to build fdk-aac with a different triplet from the rest of the libraries or build everything dynamically. Also I don't think the C runtime linkage is relevant.

Reply:
I suppose it does work because I see `fixup_bundle` copied the fdk-aac dylib into the app bundle in https://github.com/daschuer/mixxx/runs/3215138488

However my [PR to add the `he-aac` port feature](https://github.com/microsoft/vcpkg/pull/16007) upstream was just approved and I expect it to be merged soon. So I think we should figure out how to build this dynamically with the upstream port despite the rest of the packages being linked statically. This seems to be possible by hacking a special case into our custom triplet:
https://github.com/microsoft/vcpkg/blob/master/docs/users/triplets.md#per-port-customization